manifests/setting.pp

```diff
@@ -1,5 +1,6 @@
 define apt::setting (
   $setting_type,
+  $base_name  = $name,
```

Comment:
We should use `$title` here as per https://docs.puppetlabs.com/puppet/latest/reference/lang_defined_types.html#title-and-name

> $name defaults to the value of $title, but users can optionally specify a different value when they declare an instance. This is only useful for mimicking the behavior of a resource with a namevar, which is usually unnecessary. If you are wondering whether to use $name or $title, use $title.